samples/net/wifi/apsta_mode/src/main.c

```diff
@@ -41,11 +31,11 @@ static void wifi_event_handler(struct net_mgmt_event_callback *cb, uint64_t mgmt
 {
 	switch (mgmt_event) {
 	case NET_EVENT_WIFI_CONNECT_RESULT: {
-		LOG_INF("Connected to %s", WIFI_SSID);
+		LOG_INF("Connected to %s", CONFIG_WIFI_SSID);
```

Comment:
Add build asserts so that the compilation fails if user has not set the config option values that need to be set.